core/rawdb/freezer_table.go

```diff
@@ -103,6 +103,11 @@ type freezerTable struct {
 	lock   sync.RWMutex // Mutex protecting the data file descriptors
 }
 
+// NewFreezerTable opens the given path as a freezer table.
```

Comment:
?

Reply:
... What is the question?
If you're wondering why I exposed this, it's to be able to run the header-import via https://github.com/holiman/headerimport

Reply:
Ahh okay, I wondered because I didn't see it used in the code and thought it might be an artifact